app/helpers/object_link_helper.rb

```diff
@@ -60,15 +60,42 @@ def name_link(name, str = nil)
 
   # ----- links to names and records at external websites ----------------------
 
+  def ascomycete_org_name_url(name)
+    "https://ascomycete.org/Search-Results?search=#{name.text_name}"
+  end
+
+  def gbif_name_search_url(name)
+    "https://www.gbif.org/species/search?q=#{name.text_name}"
+  end
+
+  def inat_name_search_url(name)
+    "https://www.inaturalist.org/search?q=#{name.text_name}"
+  end
+
   # url for IF record
   def index_fungorum_record_url(record_id)
     "http://www.indexfungorum.org/Names/NamesRecord.asp?RecordID=#{record_id}"
   end
 
-  # url for Index Fungorum search. This is a general search.
-  # IF lacks an entry point that includes the name to be searched.
-  def index_fungorum_basic_search_url
-    "http://www.indexfungorum.org/Names/Names.asp"
+  # Use web search because IF internal search uses js form rather than a url
```

Comment:
@nimmolo: What do you think of the DuckDuckGo search results?
In particular, would other search engines yield a more useful result?
(DDG has a pop-up the first time, and may show some ads before the first real result. But other search engines also do something obnoxious.)

Reply:
@JoeCohen I tried Google's "advanced search" syntax which is a little different - and it also returned no results for your example! Good idea trying DDG, seems like the way to go.

Reply:
I would maybe consider the "tab" pattern for these, as in app/helpers/tabs. 

I've been using those to give links identifying classes for system tests. The pattern is an array: 
`if_link_tab = [link_text, url, html_options]` - there are examples that auto-generate a class name. 
then `link_to(*if_link_tab)` in the template.

Reply:
@nimmolo: Credit for the DuckDuckGo idea goes to @mo-nathan. I would never have thought of it.
Bing also returns decent results, but includes a Chat thingy that's more active, and therefore mroe annoying, than DuckDuckGo's thingy.

Reply:
@nimmolo: Re "tab" pattern. I think that's what I'm doing (and it sure makes it easier to write the code). 
Maybe I'm misunderstanding the pattern or your comment.

Reply:
Oh, yes. I missed the rest of what you were doing... 
I was looking at this on my phone and only caught the new "url" helpers in object_link_helper.